climada/hazard/centroids/centr.py

```diff
@@ -491,52 +491,98 @@ def clear(self):
         self.__init__()
 
     def append(self, centr):
-        """Append raster or points.
+        """Append centroids points.
+
+        If centr is raster it is converted to points first using
+        Centroids.set_meta_to_lat_lon. Raster information will be lost.
+
+        Note: self is modified in-place, and meta is set to {}.
 
         Parameters
         ----------
         centr : Centroids
-            If it's a raster, it needs to have the same `meta` attribute.
-            If it's of non-raster form, it's geometry needs to have the same CRS.
+            Centroids to append. The centroids need to have the same CRS.
         """
-        if self.meta and centr.meta:
-            LOGGER.debug('Appending raster')
-            if centr.meta['crs'] != self.meta['crs']:
-                raise ValueError('Different CRS not accepted.')
-            if self.meta['transform'][0] != centr.meta['transform'][0] \
-               or self.meta['transform'][4] != centr.meta['transform'][4]:
-                raise ValueError('Different raster resolutions.')
-            left = min(self.total_bounds[0], centr.total_bounds[0])
-            bottom = min(self.total_bounds[1], centr.total_bounds[1])
-            right = max(self.total_bounds[2], centr.total_bounds[2])
-            top = max(self.total_bounds[3], centr.total_bounds[3])
-            crs = self.meta['crs']
-            width = (right - left) / self.meta['transform'][0]
-            height = (bottom - top) / self.meta['transform'][4]
-            self.meta = {
-                'dtype': 'float32',
-                'width': width,
-                'height': height,
-                'crs': crs,
-                'transform': rasterio.Affine(self.meta['transform'][0], 0.0, left,
-                                             0.0, self.meta['transform'][4], top),
-            }
-            self.lat, self.lon = np.array([]), np.array([])
+        self.__dict__.update(self.union(centr).__dict__)
+
+
+    def union(self, *others):
+        """
+        Create the union of centroids from the inputs.
+
+        The centroids are combined together point by point.
+        Rasters are converted to points and raster information is lost.
```

Comment:
```suggestion
        Rasters are converted to points and this object's self raster information is lost.
```
Is this more accurate? The `*others` don't lose `meta` information, if I understand right.

Reply:
Actually, only the ouput raster information is lost. So, neither the objects self, not the others.